constraints.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --no-annotate --output-file=constraints.txt docker/prognostic_run/requirements/base.txt docker/prognostic_run/requirements/notebook.txt external/fv3fit/requirements.txt external/fv3kube/setup.py external/vcm/setup.py pip-requirements.txt workflows/post_process_run/requirements.txt workflows/prognostic_c48_run/requirements.in
```

Comment:
I don't think this existing version of `constraints.txt` was generated with the current fv3net master since the Makefile `make lock_pip` rule there does not list `external/fv3fit/requirements.txt`